packages/core/useColorMode/index.ts

```diff
@@ -77,6 +87,7 @@ export function useColorMode<T extends string = BasicColorSchema>(options: UseCo
     storageKey = 'vueuse-color-scheme',
     listenToStorageChanges = true,
     storageRef,
+    emitAuto = false,
```

Comment:
```suggestion
    emitAuto,
```

We don't need to assign the default since it's already falsy